docs/steps-elements.md

```diff
@@ -404,3 +404,9 @@ Clicks on `:elementDrag` and moves it onto `:elementDrop` while left mouse butto
 Note: This step is not working on HTML5!
 
 ---
+
+## `I confirm the recaptcha`
+
+Used to confirm recaptcha on current page.
```

Comment:
Could you specify which recaptach user want to confirm? You could also add some url to recaptcha dosc.

Reply:
I didn't see more than one ReCaptcha on one page. It's possible but I think it's gonna be less intuitive if we create such step.

Perhaps we can create a method which allows that but not in the step.

Reply:
I meant - that we should add in documentation which version of recaptacha it is - v2?